aws-pinpoint-core/src/test/java/com/amplifyframework/pinpoint/core/TargetingClientTest.kt

```diff
@@ -48,6 +48,15 @@ class TargetingClientTest {
         targetingClient = constructTargetingClient()
     }
 
+    @Test
+    fun testCurrentEndpoint() {
+        targetingClient.addAttribute("attribute", listOf("a", "b", "c"))
```

Comment:
Each unit test is clean.

Reply:
There is however duplication in other tests for constructingTargetingClient. 
```
      setup()
        targetingClient = constructTargetingClient()
```
        These can be removed from the other tests

Reply:
Not to mention they also call setup() which doesn't make a lot of sense either since its already running. FYI, this test reverts this file back to what it was before I removed this test in a previous PR, forgetting to add it back. There is some cleanup to be done but this is more of a revert commit to add a test back in.

Reply:
Yupp that is what I am saying that those 2 lines can be removed. I thought since you were already inside it you can just remove those lines as I ran them locally without those lines and it works. But we can easily create a task for that cleanup.